development/build/scripts.js

```diff
@@ -343,6 +344,7 @@ function createScriptTasks ({ browserPlatforms, livereload }) {
           : conf.INFURA_PROJECT_ID
       ),
       SEGMENT_WRITE_KEY: environment === 'production' ? SEGMENT_PROD_WRITE_KEY : SEGMENT_DEV_WRITE_KEY,
+      SEGMENT_LEGACY_WRITE_KEY: conf.SEGMENT_LEGACY_WRITE_KEY,
```

Comment:
This should only be injected in production as well I think? :thinking:

Reply:
So we won't have separate prod/dev instances for these events. What I did to handle this feedback:

1. In testing, it's undefined so we use the "noop" segment module.
2. In prod look specifically for `process.env.SEGMENT_LEGACY_WRITE_KEY` vs `conf.SEGMENT_LEGACY_WRITE_KEY`

This should mimic how it's handled for the other key with the exception being that we aren't routing production events to a different destination.